modules/persistent-storage-local-create-cr.adoc

```diff
@@ -10,7 +10,7 @@ Local volumes cannot be created by dynamic provisioning. Instead, persistent vol
 .Prerequisites
 
 * The Local Storage Operator is installed.
-* Local disks are attached to the {product-title} nodes.
+* You have a local disk that is attached to a node, is not mounted, and does not contain partitions.
```

Comment:
Could think about a nested UL.

Reply:
I agree that would be much cleaner. But the ISG doesn't like lists of that sort (that is, lists of clauses that together form a sentence). And I wasn't able to devise another combination of these elements that met the criteria. Unless you have suggestions?
@maxwelldb

Reply:
Fair enough. I've pushed the changes. Thanks!